fishbowl/api.py

```diff
@@ -787,6 +787,14 @@ def get_products_fast(self, populate_uoms=True, custom_bools=None):
         )
 
         for row in self.send_query(sql):
+            # NOTE: the PRODUCTS_SQL query selects every column from the
+            #       PRODUCT table (the P.* at the beginning) and at some
+            #       point Fishbowl has added a 'customFields' column that
+            #       seems to have a JSON blob in it... anyway, that conflicts
+            #       with how we parse out custom fields in actual XML
+            #       responses, so we get rid of it here.
```

Comment:
Interesting. Switching to the json response, in the future, could that let this exist. Then we just "magically" have access to it?

Reply:
A very solid *maybe*, It's not documented anywhere I can find, but worth keeping an eye on.